.github/workflows/build-services.yaml

```diff
@@ -2,40 +2,19 @@ name: build services
 
 on:
   push:
+    paths-ignore:
+      - '.github/**'
+      - 'digiwf-apps/**'
+      - 'docs/**'
+      - 'stack/**'
```

Comment:
Auch hier müsste es wieder aktiv sein für das Required

Reply:
Siehe Antwort weiter oben

Reply:
https://github.com/it-at-m/digiwf-core/pull/1443/files#r1524355012